[2024-04-19 19:34:12,123] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: NEO_coderhouse.notificar_objetos_cercanos manual__2024-04-19T19:34:05.921665+00:00 [queued]>
[2024-04-19 19:34:12,135] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: NEO_coderhouse.notificar_objetos_cercanos manual__2024-04-19T19:34:05.921665+00:00 [queued]>
[2024-04-19 19:34:12,136] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2024-04-19 19:34:12,137] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2024-04-19 19:34:12,137] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2024-04-19 19:34:12,154] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): notificar_objetos_cercanos> on 2024-04-19 19:34:05.921665+00:00
[2024-04-19 19:34:12,160] {standard_task_runner.py:52} INFO - Started process 610 to run task
[2024-04-19 19:34:12,163] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'NEO_coderhouse', 'notificar_objetos_cercanos', 'manual__2024-04-19T19:34:05.921665+00:00', '--job-id', '94', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpvu33tb5n', '--error-file', '/tmp/tmp831eti5j']
[2024-04-19 19:34:12,165] {standard_task_runner.py:80} INFO - Job 94: Subtask notificar_objetos_cercanos
[2024-04-19 19:34:12,226] {task_command.py:371} INFO - Running <TaskInstance: NEO_coderhouse.notificar_objetos_cercanos manual__2024-04-19T19:34:05.921665+00:00 [running]> on host 2e237166a7ec
[2024-04-19 19:34:12,289] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=LukeFo
AIRFLOW_CTX_DAG_ID=NEO_coderhouse
AIRFLOW_CTX_TASK_ID=notificar_objetos_cercanos
AIRFLOW_CTX_EXECUTION_DATE=2024-04-19T19:34:05.921665+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-04-19T19:34:05.921665+00:00
[2024-04-19 19:34:12,291] {logging_mixin.py:115} INFO - Hola
[2024-04-19 19:34:13,273] {logging_mixin.py:115} INFO - Cantidad de registros extraidos el 2024-04-19 - 18
[2024-04-19 19:34:14,227] {logging_mixin.py:115} INFO - 'ascii' codec can't encode character '\xf3' in position 132: ordinal not in range(128)
[2024-04-19 19:34:14,228] {logging_mixin.py:115} INFO - Failure
[2024-04-19 19:34:14,230] {python.py:173} INFO - Done. Returned value was: None
[2024-04-19 19:34:14,244] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=NEO_coderhouse, task_id=notificar_objetos_cercanos, execution_date=20240419T193405, start_date=20240419T193412, end_date=20240419T193414
[2024-04-19 19:34:14,271] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-04-19 19:34:14,301] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
